llvm/include/llvm/ExecutionEngine/JITLink/JITLink.h

```diff
@@ -758,6 +758,12 @@ class Section {
     return make_range(Symbols.begin(), Symbols.end());
   }
 
+  /// Returns address of the section
+  uint64_t getAddress() const {
+    assert(Symbols.size() && "Section has no symbols");
+    return (*Symbols.begin())->getAddress().getValue();
+  }
+
```

Comment:
The symbols in a section aren't currently guaranteed to be sorted. You'll need to use the `SectionRange` utility instead.